src/sentry/static/sentry/app/views/projectsDashboard/index.jsx

```diff
@@ -45,24 +47,37 @@ class Dashboard extends React.Component {
   }
 
   render() {
-    const {teams, projects, params, organization} = this.props;
-    const sortedProjects = sortProjects(projects);
+    const {teams, params, organization, loadingTeams, error} = this.props;
+
+    if (loadingTeams) {
+      return <LoadingIndicator />;
+    }
+
+    if (error) {
+      return (
+        <Alert type="error">{t('An error occurred while fetching your projects')}</Alert>
```

Comment:
Can you post a screenshot of this state?

Reply:
just posted in pr description I loosely copied the error state of organizationContext but I'll have a poke around what some other errors look like. Is there a specific style/component I should follow for this?

Reply:
Would <LoadingError> be better in this case?

Reply:
Yeah I would use `<LoadingError>`